core/validatorapi/router_internal_test.go

```diff
@@ -320,6 +320,71 @@ func TestRawRouter(t *testing.T) {
 		testRawRouter(t, handler, callback)
 	})
 
+	t.Run("get validators with post", func(t *testing.T) {
```

Comment:
can you add one more test where neither query params are provided nor request body is provided? In this case it should respond with the data of all the validators.

Reply:
My test was designed to not return any results if indexes are empty, but now fixed that to match the expected behavior and added another test.